### DISCUSSION TITLE
Feature Request - Wiki

### DISCUSSION THREAD

**Discussion by YairDaniel97** [community/community#3635] [Mobile] [answered]
We use Wikis a lot for our documentation and we just couldn't help but notice that it's the only major repo feature missing from the mobile app.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @zanaaziz – wiki support isn't on our roadmap right now, and hasn't been prioritized, but I'll add this thread to our tracking issue!

**Reply by publicarray** [reply]
@brianlovin 

At the moment links to wiki pages in the iOS app result in an unknown error. Can they me made readable in the mobile app? Thanks

**Reply by brianlovin** [reply]
@publicarray do you have a sample link I can use to reproduce the error view?

**Reply by publicarray** [reply]
@brianlovin Yea sure: https://github.com/SynoCommunity/spksrc/wiki/Frequently-Asked-Questions

(FAQ link from https://github.com/SynoCommunity/spksrc)

https://user-images.githubusercontent.com/5497998/144135346-f8d82c85-759a-4353-b69e-4b88684bff1e.MOV

**Reply by publicarray** [reply]
Ping @brianlovin

**Reply by codywalker77** [reply]
Yes currently iOS has a lot out there as far as untouched packages... I know between that and the crowdin project and trying to get through the roads up here for screening. Our days are immensely packed!!

**Reply by brianlovin** [reply]
@publicarray thank you for the link + repro video. I've flagged this to our team. In the short term, we'll bounce wiki links out to the browser.

**Reply by publicarray** [reply]
Thanks @brianlovin!

**Comment by jdhao** [comment]
May I ask where I can find your list of priorities?  Judging from the upvotes this discussion gets, I think you should add this to your priority.

 I don't want to seem to be rude, but isn't your priority the features that the users want most?

**Reply by brianlovin** [reply]
@jdhao we balance requests from our customers with lots of other inputs, like where we feel Mobile can provide unique value to customers and untether certain workflows from the desktop. Wikis could be cool, but we're currently invested in things like pull requests, issues, notifications, projects, and other more day-to-day workflows like that. All to say, we do use these upvotes and discussions as input in all of our planning 😊 

GitHub's public roadmap is here, too: https://github.com/github/roadmap

**Reply by zxpower** [reply]
Maybe its possible to split the request up in two parts - wiki as read only option could be implemented quite quick as you already parse Markdown files. Edit of the wiki could be left for later. Due to that we rely on wiki as crucial part of our documentation is quite inconvienent that its not available.

Another suggestion would be displaying wiki as separate repo which in general it is.
